llvm/lib/Support/VirtualFileSystem.cpp

```diff
@@ -324,6 +330,17 @@ ErrorOr<Status> RealFileSystem::status(const Twine &Path) {
 
 ErrorOr<std::unique_ptr<File>>
 RealFileSystem::openFileForRead(const Twine &Name) {
+  SmallString<256> RealName, Storage;
```

Comment:
Can you put this code into a static function or even a private member function so you can avoid the duplication of code.

Reply:
I've created a private function for both functions to call, thanks!